### ISSUE TITLE
Warning: Invalid prop `location` supplied the `Router`

### ISSUE THREAD

**Issue by raywan** [dferber90/webapp#6]
After starting the app and loading the the page, I get the warning:
`Warning: Failed propType: Invalid prop `location` supplied to `Router`, expected instance of `Location`.
main file`. 

It doesn't seem to affect the behaviour of the app though. 

In development:

``` bash
npm run start

> webapp-starter-pack@1.0.0 start /Users/raywan/Projects/webapp-starter-pack
> npm run clean && node webpack/dev.js


> webapp-starter-pack@1.0.0 clean /Users/raywan/Projects/webapp-starter-pack
> rimraf build/*

compiled server
server listening
Warning: Failed propType: Invalid prop `location` supplied to `Router`, expected instance of `Location`.
main file
```

In production:

``` bash
node backend.js
Warning: Failed propType: Invalid prop `location` supplied to `Router`, expected instance of `Location`.
```

**Comment by dferber90** [comment]
I am aware of this, but thank you for reporting anyways.

This is an issue within React Router, as can be seen [here](https://github.com/rackt/react-router/issues/1524). I have briefly tried debugging it, but `location` turned out to actually be an instance of `Location`. I hope this will be fixed with the 1.0 release of React Router.

**Comment by dferber90** [comment]
I am currently rewriting this from the ground up. React Router has changed the code causing this already. Will no longer be an issue after the rewrite.
